src/Team-Services-Bot.AcceptanceTests/Team-Services-Bot.AcceptanceTests.csproj

```diff
@@ -156,6 +156,9 @@
     <Compile Include="..\Common\CommonAssemblyInfo.cs">
       <Link>Properties\CommonAssemblyInfo.cs</Link>
     </Compile>
+    <Compile Include="..\Common\TestCategories.cs">
```

Comment:
I generally dislike idea of source code sharing across the projects. Why we can't create class library and have all common test classes here?